openlibrary/macros/StarRatings.html

```diff
@@ -10,27 +10,34 @@
   <meta property="bestRating" content="5"/>
   <input type="hidden" value="$edition_key" name="edition_id"/>
   <input type="hidden" value="true" name="redir"/>
-  <div class="stars">
-    $ NUM_STARS = 5
-    $for star in range(1, NUM_STARS + 1)
-      <input type="submit" value="$star" name="rating"
-        $if rating == star:
-          class="star-$star star-selected"
-        $else:
-          class="star-$star"
-        id="star-$star" />
-      <label class="star star-$star $('star-selected' if rating==star else '')"
-        $ properties = []
-        $if star == rating:
-          $ properties.append("ratingValue")
-        $if star == 1:
-          $ properties.append("worstRating")
-        $elif (star == NUM_STARS):
-          $ properties.append("bestRating")
-        $if properties:
-          property="$(' '.join(properties))"
-        for="star-$star">$star</label>
-    <span></span>
+  <div class="review">
+    <div class="stars">
+      $ NUM_STARS = 5
+      $for star in range(1, NUM_STARS + 1)
+        <input type="submit" value="$star" name="rating"
+          $if rating == star:
+            class="star-$star star-selected"
+          $else:
+            class="star-$star"
+          id="star-$star" />
+        <label class="star star-$star $('star-selected' if rating==star else '')"
+          $ properties = []
+          $if star == rating:
+            $ properties.append("ratingValue")
+          $if star == 1:
+            $ properties.append("worstRating")
+          $elif (star == NUM_STARS):
+            $ properties.append("bestRating")
+          $if properties:
+            property="$(' '.join(properties))"
+          for="star-$star">$star</label>
+      <span></span>
+    </div>
+    <div class="stats-link">
+      <a class="observations-modal-link" href="javascript:;" data-context="{&quot;username&quot;: &quot;mekBot&quot;, &quot;work&quot;: &quot;/works/OL516825W&quot;, &quot;id&quot;: &quot;stats&quot;}">
+        <img src="/static/images/icons/review.png" class="review-btn">
+      </a>
+    </div>
```

Comment:
```suggestion
    $if username:
      $:macros.ObservationsModal(work, '', 'img-link', classes='stats-link', image_source="/static/images/icons/review.png", img_classes='review-btn')
    $else:
      <div class="stats-link">
        <a class="observations-modal-link" href="/account/login">
          <img src="/static/images/icons/review..png" class="review-btn">
        </a>
      </div>
```
ObservationModals require the username of a logged in patron to render properly.  This should redirect to account login if patron clicks while not logged in (note that login link in suggestion is missing the redirect link).

ObservationModal call with image source requires #5578 to be merged.